EssentialsGeoIP/src/com/earth2me/essentials/geoip/EssentialsGeoIPPlayerListener.java

```diff
@@ -82,6 +81,16 @@ private void delayedJoin(Player player) {
                 sb.append(response.getCountry().getName());
             }
         } catch (AddressNotFoundException ex) {
+
+            if (checkIfLocal(address)) {
+                for (Player online : player.getServer().getOnlinePlayers()) {
+                    User user = ess.getUser(online);
+                    if (user.isAuthorized("essentials.geoip.show")) {
+                        user.sendMessage(tl("geoipCantFind", u.getDisplayName()));
```

Comment:
So we hit this condition when address is a local address - isn't the message for this a bit misleading? Sure we can't find a country but I don't think showing this for locally originating IPs is the best option. This just doesn't seem like the right logic here.

Reply:
These changes are not relevant to this PR. They were just an issue because I'm new to working with branches in git and sort of messed up.